views/static.html

```diff
@@ -17,7 +17,7 @@
       </li>
       {% endif %}
       <li class="breadcrumb_item">
-        <a class="breadcrumb_link" href="#" aria-current="page">{{ title | replace('&amp;', '&') or __(slug)}}</a>
+        <a class="breadcrumb_link" href="#" aria-current="page">{{ title  | safe or __(slug)}}</a>
```

Comment:
@gavram this is great. Do you know why are we using translation for `slug` here? May be it shouldn't be translatable.

Reply:
I removed `slug` without breaking up functionality or producing errors. There is no need for translation because blog posts language will be always Danish or if it will be not this option can be added.